curriculum/challenges/english/25-front-end-development/workshop-fcc-authors-page/641d9a19bff38d34d5a5edb8.md

```diff
@@ -8,36 +8,39 @@ demoType: onLoad
 
 # --description--
 
-All the HTML and CSS for this project has been provided for you. You can take a look at the two files to familiarize yourself with them.
+In this workshop, you will learn how to work with the `fetch` API to load data from an external source and display it on a page filled with freeCodeCamp authors.
 
-Start by getting the `#author-container` and `#load-more-btn` elements with the `.getElementById()` method. Assign them to the variables `authorContainer` and `loadMoreBtn`, respectively.
+All the HTML and CSS for this workshop has been provided for you. You can take a look at the two files to familiarize yourself with them.
 
-The variables will not change, so use `const` to declare them.
+When you are ready, start by accessing the `#author-container` and `#load-more-btn` elements. Remember you can use the `getElementById` or `querySelector` methods for this.
+
+Then assign these elements to variables  called `authorContainer` and `loadMoreBtn`, respectively.
 
 # --hints--
 
-You should use `document.getElementById()` to get the `#author-container` element.
+You should have a variable called `authorContainer`.
 
 ```js
-assert.match(code, /document\.getElementById\(\s*('|"|`)author\-container\1\s*\)/);
+assert.isDefined(authorContainer, 'The variable authorContainer should be defined');
 ```
 
-You should assign the `#author-container` element to the variable `authorContainer`. Don't forget to use `const` to declare the variable.
+You should assign the `#author-container` element to the `authorContainer` variable.
 
 ```js
-assert.match(code, /const\s+authorContainer\s*\=\s*document\.getElementById\(\s*('|"|`)author\-container\1\s*\)/);
+assert.strictEqual(authorContainer, document.querySelector('#author-container'), 'authorContainer should be assigned the #author-container element');
 ```
 
-You should use `document.getElementById()` to get the `#load-more-btn` element.
+You should have a variable called `loadMoreBtn`.
 
 ```js
-assert.match(code, /document\.getElementById\(\s*('|"|`)load\-more\-btn\1\s*\)/);
+assert.isDefined(loadMoreBtn, 'The variable loadMoreBtn should be defined');
 ```
 
-You should assign the `#load-more-btn` element to the variable `loadMoreBtn`. Don't forget to use `const` to declare the variable.
+You should assign the `#load-more-btn` element to the `loadMoreBtn` variable.
+
 
 ```js
-assert.match(code, /const\s+loadMoreBtn\s*\=\s*document\.getElementById\(\s*('|"|`)load\-more\-btn\1\s*\)/);
+assert.strictEqual(loadMoreBtn, document.querySelector('#load-more-btn'), 'loadMoreBtn should be assigned the #load-more-btn element');
```

Comment:
```suggestion
assert.strictEqual(loadMoreBtn, document.querySelector('#load-more-btn'));
```

Reply:
thanks for your feedback @jdwilkin4 
latest commit resolves all the suggestions